pyquil/paulis.py

```diff
@@ -324,6 +324,24 @@ def from_list(cls, terms_list, coefficient=1.0):
         pterm.coefficient = complex(coefficient)
         return pterm
 
+    def pauli_string(self, min_index=None, max_index=None):
```

Comment:
These default arguments don't make sense to me. Document and/or fix :)

Reply:
Sorry, wasn't done!

Reply:
my bad!